### DISCUSSION TITLE
Orchard training demo couldn't start

### DISCUSSION THREAD

**Discussion by FireEagle97** [OrchardCMS/OrchardCore#15103] [Q&A] [answered]
Hi,
I tried to start the training demo project after following the steps and installing Node.js and PowerShell 7 but I got more than 40 errors.
What I am missing here?
![err](https://github.com/OrchardCMS/OrchardCore/assets/62710297/7dca4c07-99da-4b2c-8c8e-dd5a5e941986)

**Comment by nicpitsch** [comment]
It could be the missing git submodules, that is the 3th point of [Prerequisites](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions#prerequisites-and-getting-started).

**Reply by FireEagle97** [reply]
I ran the command multiple times I don't know if it made any effect.

**Reply by nicpitsch** [reply]
Did you try using the submodules switch from the beginning: `git clone https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions.git --recurse-submodules`?

**Reply by FireEagle97** [reply]
Do I need to clone the Extensions repo inside the training module repo?

**Reply by nicpitsch** [reply, marked as answer]
No. The instruction is to clone the *Open-Source Orchard Core Extensions* repo using `git clone https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions.git --recurse-submodules`.

After that you may proceed with the steps 2. to 4. as explained in [*Orchard-Training-Demo-Module* readme](https://github.com/Lombiq/Orchard-Training-Demo-Module).

I mentioned the *Orchard-Training-Demo-Module* in discussion [#15099](https://github.com/OrchardCMS/OrchardCore/discussions/15099) in connection with Module development. If that's your interest, you may find there some more information.

**Reply by FireEagle97** [reply]
thank you it now running and I will follow the steps in the StartLearningHere.md

**Comment by hishamco** [comment]
@FireEagle97 please move this to the Lombiq repository, @Piedone I think you have permission to move it there

**Reply by Piedone** [reply]
It's not possible to transfer discussions across organizations but @FireEagle97 feel free to open an issue here: https://github.com/Lombiq/Orchard-Training-Demo-Module/issues. However, for now, my tips would be the same as Nicolo's. Also, the errors you show are an effect, rather than the cause, of some problem. I'd suggest starting anew from https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions#prerequisites-and-getting-started, building the solution, and if it doesn't work, include the full build output (the Output window in VS), not the Error List, here.

**Reply by hishamco** [reply]
> It's not possible to transfer discussions across organizations

I thought it was possible because you are a core member for both, anyway, @FireEagle97 please follow the @Piedone suggestion please
